src/conf_mode/snmp.py

```diff
@@ -86,7 +86,7 @@ def get_config(config=None):
     return snmp
 
 def verify(snmp):
-    if not snmp:
+    if 'deleted' in snmp:
```

Comment:
Should we use `if not snmp or 'deleted' in snmp`?
I didn't check

Reply:
SNMP key is always present